### ISSUE TITLE
Include how to organize dependencies, middleware, and routes if using the Slim skeleton project

### ISSUE THREAD

**Issue by jbulava** [DavidePastore/Slim-Config#2]
I noticed the README example includes the dependencies, middleware, and routing all within one file.  It may be worth adding a note for anyone using the Slim skeleton project that these should be separated into `src/dependencies.php`, `src/middleware.php`, and `src/routes.php`.

**Comment by DavidePastore** [comment]
Yes, you're right! Feel free to create a PR for this if you already have something in mind. :+1:
